### PR TITLE
Moderate Damage Examination

### PR DESCRIPTION
**Pull request by Robustin**
Ok so the last one only worked on carbons who weren't humans, woops. See: https://github.com/tgstation/tgstation/pull/32707

🆑 Robustin
tweak: Damage examinations now include a "moderate" classification. Before minor was <30 and severe was anything 30 or above. Now minor is <25, moderate is 25 to <50, and severe is 50+.
/🆑

### CODE REVIEW

**Comment by DoctoreMad** [comment]
>Damaging people because they looked at you

This would definitely become the new meta if it wasn't for the fact that it was just me misreading the PR title.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -188,24 +188,30 @@
 		msg += "[t_He] [p_do()]n't seem all there.\n"
 
 	if(temp)
-		if(temp < 30)
+		if (temp < 25)
 			msg += "[t_He] [t_has] minor bruising.\n"
+		else if (temp < 50)
+			msg += "[t_He] [t_has] <b>moderate</b> bruising!\n"
```
Comment (Thunder12345) [inline code/modules/mob/living/carbon/human/examine.dm]: Why is moderate bolded?

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -188,24 +188,30 @@
...
 		else
 			msg += "<B>[t_He] [t_has] severe bruising!</B>\n"
 
 	temp = getFireLoss()
 	if(temp)
-		if(temp < 30)
+		if (temp < 25)
 			msg += "[t_He] [t_has] minor burns.\n"
+		else if (temp < 50)
+			msg += "[t_He] [t_has] <b>moderate</b> burns!\n"
```
Comment (Thunder12345) [inline code/modules/mob/living/carbon/human/examine.dm]: Ditto

File: code/modules/mob/living/carbon/human/examine.dm [hunk 3]
```diff
@@ -188,24 +188,30 @@
...
 		else
 			msg += "<B>[t_He] [t_has] severe burns!</B>\n"
 
 	temp = getCloneLoss()
 	if(temp)
-		if(temp < 30)
+		if(temp < 25)
 			msg += "[t_He] [t_has] minor cellular damage.\n"
+		else if (temp < 50)
+			msg += "[t_He] [t_has] <b>moderate</b> cellular damage!\n"
```
Comment (Thunder12345) [inline code/modules/mob/living/carbon/human/examine.dm]: Ditto

**Comment by Robustin** [comment]
because its the middle ground between non-bolded minor damage and fully bolded severe damage.

**Comment by AnonymousNow** [comment]
Readability is always useful.

**Comment by Bawhoppen** [comment]
Having more obfuscated information can be better...

**Comment by fludd12** [comment]
Would it be possible to put this in mob level as well, and have it use maxhealth*0.75 through maxhealth*0.5 in either case?

**Comment by Robustin** [comment]
Anythings possible but thats different code and people here always find something to bitch about.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 4]
```diff
@@ -188,24 +188,30 @@
 		msg += "[t_He] [p_do()]n't seem all there.\n"
 
 	if(temp)
-		if(temp < 30)
+		if (temp < 25)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/examine.dm]: Stahp.

Comment (Xhuis) [reply code/modules/mob/living/carbon/human/examine.dm]: Spaces between ifs and parentheses are a crime against humanity.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 5]
```diff
@@ -188,24 +188,30 @@
 		msg += "[t_He] [p_do()]n't seem all there.\n"
 
 	if(temp)
-		if(temp < 30)
+		if(temp < 25)
 			msg += "[t_He] [t_has] minor bruising.\n"
+		else if (temp < 50)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/examine.dm]: Space between the `else if` and the `(`

Not good.

Comment (JJRcop) [reply code/modules/mob/living/carbon/human/examine.dm]: In tg's Contributing.md we don't care about this

**Comment by JJRcop** [comment]
https://github.com/tgstation/tgstation/blob/3ac9d24cb0e534cb0a0a71008c40ffab0934e49d/.github/CONTRIBUTING.md#L273

**Comment by Robustin** [comment]
This is literally a fix for a merged PR, its silly for it to be languishing.

**Pull request merged**